MLAPI/Data/NetworkedVarMeta.cs

```diff
@@ -0,0 +1,30 @@
+﻿using MLAPI.Data.Transports;
+
+namespace MLAPI.Data
+{
+    public class NetworkedVarSettings
+    {
+        public NetworkedVarPermission WritePermission = NetworkedVarPermission.ServerOnly;
+        public NetworkedVarPermission ReadPermission = NetworkedVarPermission.Everyone;
+        public NetworkedVarPermissionsDelegate WritePermissionCallback = null;
+        public NetworkedVarPermissionsDelegate ReadPermissionCallback = null;
+        public bool SendOnChange = false;
+        public float SendDelay = 0.1f;
```

Comment:
If SendOnChange is true, it will send everytime it's dirty (but only on the SendTickrate).

But I'm assuming you want the SendOnChange to default to true?